service/config.go

```diff
@@ -13,6 +13,10 @@ type Config struct {
 	Listener net.Listener
 	// ProcessArgs are the arguments to launch a new process.
 	ProcessArgs []string
+	// Wd is working directory of the new process. This field is used only when
+	// ProcessArgs is specified.
```

Comment:
Technically this comment is correct but it sounds pretty weird, I'd suggest: `s/only when ProcessArgs is specified/only when launching a new process./`